src/pages/layout/Search/Index.tsx

```diff
@@ -101,6 +106,24 @@ export default function HeaderSearch() {
     }
   };
 
+  const handleEnterPress = async () => {
+    try {
+      await getAccount({address: inputValue}, state.network_value);
```

Comment:
Can you start both requests simultaneously instead of sequentially? e.g. `const accountPromise = getAccount(); const transactionPromise = getTransaction(); ... await accountPromise; ... await transactionPromise;` This will reduce latency if it's a transaction.